go-client/TestSetupAbstraction.go

```diff
@@ -164,11 +164,16 @@ type ConnectionInfo struct {
 }
 
 func (testSetup *TestSetupAbstraction) CreateConnection() *sql.DB {
+	return testSetup.CreateConnectionWithConfig(true)
+}
+
+func (testSetup *TestSetupAbstraction) CreateConnectionWithConfig(autocommit bool) *sql.DB {
 	connectionDetails := testSetup.getConnectionInfo()
 	connection, err := sql.Open("exasol", exasol.NewConfig(connectionDetails.User, connectionDetails.Password).
 		Port(connectionDetails.Port).
 		Host(connectionDetails.Host).
 		ValidateServerCertificate(false).
+		Autocommit(autocommit).
```

Comment:
Another, more flexible option would be to provide a method that returns a new, populated config with user, password, host & port. Then the user has more freedom to configure the connection.
We can add this next time.